### PR TITLE
libemf2svg 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by ErezVolk**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/libemf2svg.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  sha256 "ad48d2de9d1f4172aca475d9220bbd152b7280f98642db561ee6688faf50cd1e"
+  license "GPL-2.0-only"
+
+  depends_on "argp-standalone" => :build
+  depends_on "cmake" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "libpng"
+
+  resource("testdata") do
```
Comment (SMillerDev) [inline Formula/libemf2svg.rb:14]: This stops our automation from suggesting updates with every release
```suggestion
  resource("homebrew-testdata") do
```

File: Formula/libemf2svg.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  resource("homebrew-testdata") do
+    url "https://github.com/kakwa/libemf2svg/raw/1.1.0/tests/resources/emf/test-037.emf"
+    sha256 "d2855fc380fc3f791da58a78937af60c77ea437b749702a90652615019a5abdf"
+  end
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+  end
```
Comment (chenrui333) [inline Formula/libemf2svg.rb:23]:
```suggestion
  resource "homebrew-testdata" do
    url "https://github.com/kakwa/libemf2svg/raw/1.1.0/tests/resources/emf/test-037.emf"
    sha256 "d2855fc380fc3f791da58a78937af60c77ea437b749702a90652615019a5abdf"
  end

  def install
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
  end
```

Comment (chenrui333) [reply Formula/libemf2svg.rb:23]: Just some code style suggestion, and this cmake steps are kind of de-facto now

```
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Comment (ErezVolk) [reply Formula/libemf2svg.rb]: Thank you!

**Review by chenrui333 — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@ErezVolk, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**